crates/context_aware_config/src/api/custom_types/types.rs

```diff
@@ -0,0 +1,22 @@
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct TypeTemplateRequest {
+    pub type_schema: Value,
+    pub type_name: String,
+}
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct CustomTypeResponse {
+    pub type_schema: Value,
+    pub type_name: String,
+    pub created_at: String,
+    pub last_modified: String,
+    pub created_by: String,
+}
+#[derive(Debug, Clone, Deserialize)]
+pub struct QueryFilters {
+    pub count: Option<i64>,
```

Comment:
@Datron can we make both i32 , as in code this allowing multiplying two 64 bit number which is resulting in thread panic

Reply:
@pratikmishra356 I didn't get you, why would i64 cause thread panic?

Reply:
in handler you are multiplying count and page , now if we send those two number greater than 2^32 it is causing thread panic, you can this

Reply:
Okay, what is the worst case here? The thread panics, request is not completed right? Which is okay. This isn't an attack vector since actix will bring back the thread worker.

Reply:
Also diesel's limit and offset functions expect i64